module-bsp/board/rt1051/puretx/bsp/battery_charger/battery_charger.cpp

```diff
@@ -544,7 +622,7 @@ namespace bsp::battery_charger
 
         // Short time to synchronize after configuration
         vTaskDelay(pdMS_TO_TICKS(100));
-        getBatteryLevel();
+        getBatteryLevel().value();
```

Comment:
this does not look nice TBH :P If we get the battery level we shouldn't discard its return value - or move the register read function somewhere else.

Reply:
Done